scripts/send_to_bot.py

```diff
@@ -1,41 +1,94 @@
-import json
+#!/usr/bin/env python3
+
 import os
 import sys
-try:
-  import urllib.request as urllib2
-except ImportError:
-  import urllib2
-
-URL = 'https://tldr-bot.starbeamrainbowlabs.com/'
-
-def post_comment(pr_id, comment_body):
-  # Constructing the url
-  req = urllib2.Request(URL,
-                       json.dumps({'body': comment_body, 'pr_id': pr_id }),
-                       {'Content-Type': 'application/json'})
-  # Making the request
-  f = urllib2.urlopen(req)
-  if f.getcode() != 200:
-    print(f.read())
-
-
-# Get the environment variables
-PR_NUMBER = os.environ.get('TRAVIS_PULL_REQUEST')
-BUILD_ID = os.environ.get('TRAVIS_BUILD_ID')
-
-# Read the test result output from stdin
-test_result = sys.stdin.read().strip()
-# Populate the template text
-comment = (
-"The [build]"
-"(https://travis-ci.org/tldr-pages/tldr/builds/{build_id})"
-" for this PR has failed with the following error(s):"
-"\n```\n"
-"{comment_body}"
-"\n```\n"
-"Please fix the error(s) and push again."
-).format(build_id=BUILD_ID, comment_body=test_result)
-
-# If it's a PR, post a comment on it
-if PR_NUMBER != "false":
-  post_comment(PR_NUMBER, comment)
+import json
+import urllib.request
+
+BOT_URL = 'https://tldr-bot.starbeamrainbowlabs.com'
+
+COMMENT_ERROR="""
+The [build](https://travis-ci.org/tldr-pages/tldr/builds/{build_id})
+for this PR failed with the following error(s):
+
+```
+{content}
+```
+
+Please fix the error(s) and push again.
+"""
+
+COMMENT_CHECK="""
+Hello! I've noticed something unusual when checking this PR:
+
+{content}
+
+Is this intended? If so, just ignore this comment. Otherwise, please
+double-check the commits.
+"""
+
+################################################################################
+
+def post_comment(pr_id, body, once):
+  if once:
+    endpoint = BOT_URL + '/comment/once'
+  else:
+    endpoint = BOT_URL + '/comment'
+
+  headers = {'Content-Type': 'application/json'}
+  data = json.dumps({'pr_id': pr_id, 'body': body})
+  req = urllib.request.Request(endpoint, data.encode(), headers)
```

Comment:
Now I remember why I was talking about not using `requests` in the tldr bot script. It was here that I did not use `requests` :stuck_out_tongue_closed_eyes: